src/internal/proc.js

```diff
@@ -403,6 +403,13 @@ export default function proc(
     const cancelPromise = promise[CANCEL]
     if(typeof cancelPromise === 'function') {
       cb.cancel = cancelPromise
+    } else {
```

Comment:
I did it this way cause there wasn't really a good scope to put the local abort variable.